packages/graphql-extensions/src/index.ts

```diff
@@ -47,7 +47,9 @@ export class GraphQLExtension<TContext = any> {
     executionArgs: ExecutionArgs;
   }): EndHandler | void;
 
-  public willSendResponse?(o: { graphqlResponse: GraphQLResponse }): void;
+  public willSendResponse?(o: {
```

Comment:
Hmmm, why are we passing in and returning `{ graphqlResponse: GraphQLResponse }` instead of a single object? If this is about being able to set HTTP headers, maybe we can find another way to parametrize `GraphQLResponse` instead.

Reply:
It's to leave room for future changes. I think it's definitely something we can revisit after 2.0